src/content/getusermedia/desktopcapture/extension/content-script.js

```diff
@@ -17,7 +17,7 @@ port.onMessage.addListener(function(msg) {
 });
 
 window.addEventListener('message', function(event) {
-	// We only accept messages from ourselves
```

Comment:
I have no idea why Travis complained to me about this since I didn't change this file, but I though I might as well fix this while I was at it.